submission/proxy_test.go

```diff
@@ -121,3 +122,48 @@ func TestProxyRefreshRootsErr(t *testing.T) {
 		}
 	}
 }
+
+func TestCTPolicyTypeFromString(t *testing.T) {
+	testCases := []struct {
+		name      string
+		from      string
+		wantPlc   CTPolicyType
+		errRegexp *regexp.Regexp
+	}{
+		{
+			name:      "chrome",
+			from:      "chrome",
+			wantPlc:   ChromeCTPolicy,
+			errRegexp: nil,
+		},
+		{
+			name:      "apple",
+			from:      "apple",
+			wantPlc:   AppleCTPolicy,
+			errRegexp: nil,
+		},
+		{
+			name:      "noValue",
+			from:      "crome",
+			wantPlc:   ChromeCTPolicy,
+			errRegexp: regexp.MustCompile("CTPolicyTypeFromString does not support value"),
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			gotPlc, err := CTPolicyTypeFromString(tc.from)
+			if gotErr, wantErr := err != nil, tc.errRegexp != nil; gotErr != wantErr {
+				var unwantedErr string
+				if gotErr {
+					unwantedErr = fmt.Sprintf(" (%q)", err)
+				}
+				t.Errorf("Got error = %v%s, expected error = %v", gotErr, unwantedErr, wantErr)
+			} else if tc.errRegexp != nil && !tc.errRegexp.MatchString(err.Error()) {
+				t.Errorf("Error %q did not match expected regexp %q", err, tc.errRegexp)
+			}
+			if gotPlc != tc.wantPlc {
+				t.Errorf("CtPolicyTypeFromStrin(%q) want %v, got %v", tc.from, tc.wantPlc, gotPlc)
```

Comment:
How about `CTPolicyTypeFromString: got %v, want %v`? Usually we put "want" to the end.